redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -585,136 +682,10 @@ inline void
-                            BMCWEB_LOG_ERROR << "Failed to get "
-                                             << "the HardwareIsolationReason "
-                                             << "message registry to add "
-                                             << "in the condition";
-                            messages::internalError(aResp->res);
-                            return;
-                        }
-
-                        // Prepare MessageArgs as per defined in the
-                        // MessageRegistries
-                        std::vector<std::string> messageArgs{*msgPropVal};
-
-                        // Fill the "msgPropVal" as reason
-                        std::string message = msgReg->message;
-                        int i = 0;
-                        for (const std::string& messageArg : messageArgs)
-                        {
-                            std::string argIndex = "%" + std::to_string(++i);
-                            size_t argPos = message.find(argIndex);
-                            if (argPos != std::string::npos)
-                            {
-                                message.replace(argPos, argIndex.length(),
-                                                messageArg);
-                            }
-                        }
-                        // Severity will be added based on the event
-                        // object property
-                        condition["Message"] = message;
-                        condition["MessageArgs"] = messageArgs;
-                        condition["MessageId"] =
-                            "OpenBMC.0.2.HardwareIsolationReason";
-                    }
-                    else if (property.first == "Severity")
-                    {
-                        const std::string* severity =
-                            std::get_if<std::string>(&property.second);
-                        if (severity == nullptr)
-                        {
-                            BMCWEB_LOG_ERROR
-                                << "Failed to get the Severity "
-                                << "from object: " << hwStatusEventObj;
-                            messages::internalError(aResp->res);
-                            return;
-                        }
-
-                        // we have only one condition
-                        nlohmann::json::json_pointer severityPropPath =
-                            "/Status/Conditions/0/Severity"_json_pointer;
-                        if (!setSeverity(aResp, hwStatusEventObj,
-                                         severityPropPath, *severity))
-                        {
-                            // Failed to set the severity
-                            return;
-                        }
-                    }
-                }
-                },
-                objType[0].first, hwStatusEventObj,
-                "org.freedesktop.DBus.Properties", "GetAll", "");
+                // TODO
```

Comment:
why TODO? anything is spending? if yes, please include what needs todo (+ issue id)

Reply:
Sorry. That was a note to myself when it was WIP. Now it is done. I will remove it